src/js/components/Select/SelectContainer.js

```diff
@@ -51,12 +51,19 @@ const ClearButton = forwardRef(
         ref={ref}
         onClick={onClear}
         focusIndicator={false}
+        plain
         {...passThemeFlag}
         {...rest}
       >
-        <Box {...theme.select.clear.container} align={align}>
-          <Text {...theme.select.clear.text}>{buttonLabel}</Text>
-        </Box>
+        {({ hover }) => (
+          <Box
+            {...theme.select.clear.container}
```

Comment:
Won't this accidentally put a `hover=[object, object]` on the Box?

Reply:
![image](https://github.com/user-attachments/assets/fd3a07f3-9cfa-45f9-afd5-31be59d1d1a0)

Reply:
It wasn't passing it to the DOM so originally I was okay with that, but I've added one more step to avoid it being passed in React props as well:

<img width="1190" alt="Screenshot 2024-10-24 at 2 22 42 PM" src="https://github.com/user-attachments/assets/3cd18740-ad52-4c7f-a7aa-e644d75cfbfb">